### PR TITLE
updated requirements

### PR DESCRIPTION
**Pull request by Plamen5kov**
@ikoevska

### CODE REVIEW

File: requirements.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ position: 2
 The recommended ways to create NativeScript Applications is either through the [Command-Line Interface (CLI)](https://github.com/NativeScript/nativescript-cli) or through the [Telerik AppBuilder](http://www.telerik.com/appbuilder).
 
 # Supported API Levels
-The Android Runtime is built against [API level 17](http://developer.android.com/about/versions/android-4.2.html). The currently available [metadata](metadata/overview.md) is based on API level 17. Additionally, using the [metadata-generator](../metadata/generator.md) tool you may generate your own metadata for the latest [API level 21](http://developer.android.com/about/versions/android-5.0.html).
+The Android Runtime is built against [API level 17](http://developer.android.com/about/versions/android-4.2.html). The currently available [metadata](metadata/overview.md) is based on the set compile sdk. You can set the compile sdk through nativescript cli. Additionally, using the [metadata-generator](metadata/generator.md) tool you may generate your own metadata for the latest [API level 21](http://developer.android.com/about/versions/android-5.0.html).
```
Comment (ikoevska) [inline requirements.md]: What is this "set compile sdk"? Probably go with "is based on the target SDK for your project. You can set the target SDK in the NativeScript SDK when you add the Android platform to your project (`$ tns platform add android --sdk <Value>`). You can also generate your own metadata for the latest [API level 21](http://developer.android.com/about/versions/android-5.0.html) with the [metadata-generator](metadata/generator.md).

Comment (Plamen5kov) [inline requirements.md]: The compile sdk is the one used when the project is being built.
The metadata is generated based on the compile sdk, not the target sdk. Cli can pass different values for target and compile sdk.

Comment (ikoevska) [inline requirements.md]: How is the compile SDK set?

Comment (Plamen5kov) [inline requirements.md]: Either through gradle or through cli.
Cli can pass parameters to build.gradle file and tell it to build project using this or that compile or target sdk.

Comment (ikoevska) [inline requirements.md]: In your text, you need to point the default compile SDK and a clear way for setting it, as both are currently unclear. You need to specify the command that does the trick.

**Pull request merged**